test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -176,7 +175,7 @@ public String getErrorMessage() {
 
     TestShell(String user, String rootPass, String instanceName, String zookeepers, File configFile)
         throws IOException {
-      ClientInfo info = Accumulo.newClient().from(configFile.getAbsolutePath()).info();
+      ClientInfo info = ClientInfo.from(configFile.toPath());
```

Comment:
Are these equivalent? I'm not 100% sure the relationship between File and Path, and when to use one or the other, or how absolute/relative paths get resolved in either.

Reply:
They should be. I ran this IT and it passes.